src/indexes.rs

```diff
@@ -1798,6 +1798,10 @@ pub struct IndexStats {
     pub number_of_documents: usize,
     pub is_indexing: bool,
     pub field_distribution: HashMap<String, usize>,
+    pub raw_document_db_size: usize,
+    pub avg_document_size: usize,
+    pub number_of_embedded_documents: usize,
+    pub number_of_embeddings: usize,
```

Comment:
lets add doc comments too

```suggestion
    /// Total number of documents in an index
    pub number_of_documents: usize,
    /// If `true`, the index is still processing documents and attempts to search will result in undefined behavior
    pub is_indexing: bool,
    /// Shows every field in the index along with the total number of documents containing that field in said index
    pub field_distribution: HashMap<String, usize>,
    /// Storage space claimed by all documents in the index in bytes
    pub raw_document_db_size: usize,
    /// Total size of the documents stored in an index divided by the number of documents in that same index
    pub avg_document_size: usize,
    /// Total number of documents with at least one embedding
    pub number_of_embedded_documents: usize,
    /// Total number of embeddings in an index
    pub number_of_embeddings: usize,
```